### PR TITLE
Pushing code to Lesson 6 assignment

### PR DESCRIPTION
**Pull request by grkumar82**
pushing code for lesson 6 assignment

### CODE REVIEW

File: students/ravi_g/Lesson6/test_mailroom.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    assert m2.add_new_donors('Vicky V', 'one') == 'Invalid entry for donation amount'
+    # This test validates since donation amount is not valid entry
+    assert m2.add_new_donors('Vesper V', -1) == 'Invalid entry for donation amount'
+
+# These tests check if there are letters (files) generated in the folder
+def test_file_check():
+    for k,v in m2.donors.items():
+        filename1 = r"C:/Users/ravigant/UW_Python/"+ k.split(' ')[0] + "/"
+        filename1 += '20201014.txt' # date of when files ran
+        assert os.path.isfile(filename1)
```
Comment (charlesrobison) [inline students/ravi_g/Lesson6/test_mailroom.py:46]: Getting one failure in these tests

File: students/ravi_g/Lesson07/run_html_render.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+item.append( hr.A("http://google.com", "link") )
+item.append("to google")
+
+list.append(item)
+
+body.append(list)
+
+page.append(body)
+
+render_page(page, "test_html_output8.html")
```
Comment (charlesrobison) [inline students/ravi_g/Lesson07/run_html_render.py:231]: Unable to run this

**Review by charlesrobison — APPROVED** [review]
Please take another look at html render as I'm unable run it without errors.  Also, not all of your tests are passing for mailroom and is missing the option to list donors when adding new donations.  Let me know if you have questions.  Please submit a new PR if you would like another review.

Ready to merge

**Comment by grkumar82** [comment]
Hello Charles,

I will double-check and get back to you. I see that my score for html
render is below average, can I re-submit and get a new score? Thank you.

**Comment by natasha-aleksandrova** [comment]
@grkumar82 absolutely - create a new PR when you're ready.

**Comment by grkumar82** [comment]
Looking for any feedback on code for the last assignment before I submit in
the course material -

cli:
https://github.com/grkumar82/SP_Online_PY210/blob/master/students/ravi_g/lesson09/cli.py
donor_models:
https://github.com/grkumar82/SP_Online_PY210/blob/master/students/ravi_g/lesson09/donor_models.py
testing:
https://github.com/grkumar82/SP_Online_PY210/blob/master/students/ravi_g/lesson09/test_oo_mailroom.py

Thank you
Ravi.

**Pull request merged**